### ISSUE TITLE
llfi.index.map.txt maps N/A to LLVM IR instructions during tracing

### ISSUE THREAD

**Issue by abrahamchan** [DependableSystemsLab/LLTFI#37]
When tracing is enabled, LLTFI generates a file named `llfi.index.map.txt`. This file should contain the mapping between the LLVM IR id with the original line number in the code. However, the LLVM IR instructions map to `line_N/A` instead.
Additionally, we are lacking any information in the README or Wiki to explain what this file does.

Example file contents for `llfi.index.map.txt`
```
llfiID_34 line_N/A
llfiID_33 line_N/A
llfiID_32 line_N/A
llfiID_31 line_N/A
```

**Comment by abrahamchan** [comment]
This issue can be manually solved by adding a `-g` option to the compiler flags when compiling the test program to LLVM IR.

This allows the mapping file to correctly display the original line numbers.

Example file contents for `llfi.index.map.txt` after program is built with `-g` option.
```
llfiID_39 line_16
llfiID_38 line_16
llfiID_37 line_15
llfiID_36 line_15
```

However, the `GenerateMakefile` script needs to be updated to supply the `-g` option by default to enable this.

**Comment by abrahamchan** [comment]
This issue has been fixed by #38.
